transports/websys-webtransport/src/lib.rs

```diff
@@ -0,0 +1,45 @@
+
+use web_sys::WebTransport;
+use libp2p_core::Transport;
+
+pub struct WebTransportTransport;
```

Comment:
```suggestion
pub struct Transport;
```

See #2217 for the naming convention we are trying to follow.